### PR TITLE
Refs [Task][#439614][component change password UI]

### PR DESCRIPTION
**Pull request by nghianv-1868**

### CODE REVIEW

File: client/src/views/ProfilePage.vue [hunk 1]
```diff
@@ -265,6 +302,43 @@ export default {
...
+      ruleChangePass: {
+        oldPass: [
+          {
+            required: true,
+            message: "current password is required",
+            trigger: "blur"
+          },
+          {
+            min: 6,
+            message: "current password must be at least 6 characters",
```
Comment (ntquang22298) [inline client/src/views/ProfilePage.vue]: must contain

Comment (nghianv-1868) [reply client/src/views/ProfilePage.vue]: google dịch nó dịch thế em ạ :D :D

File: client/src/views/ProfilePage.vue [hunk 2]
```diff
@@ -356,6 +430,27 @@ export default {
...
       });
+    },
+    submitChangePassword(formName) {
+      let self = this;
+      this.$refs[formName].validate(async valid => {
+        if (valid) {
+          this.fullscreenLoading = true;
+          let data = await this.changePass(self.changePassword);
+          if (data.success) {
+            Message.success("change password Success!");
```
Comment (dotrungkien) [inline client/src/views/ProfilePage.vue]: `Your password has been changed successfully.`

File: client/src/views/ProfilePage.vue [hunk 3]
```diff
@@ -356,6 +430,27 @@ export default {
...
+          } else {
+            Message.error(data.msg);
+          }
+          self.fullscreenLoading = false;
+        } else {
+          console.log("change password error submit!!");
```
Comment (dotrungkien) [inline client/src/views/ProfilePage.vue]: `Something went wrong, please try again.`

**Pull request merged**